cloud-infrastructure/storage/block-storage/README.md

```diff
@@ -0,0 +1,71 @@
+# Block Storage
+
```

Comment:
I would recommend putting this link under 'Useful Links'

Reply:
done